electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -1,7 +1,8 @@
 from abc import ABC, abstractmethod
+from collections.abc import Sequence
```

Comment:
I guess they are supporting both from `collections` and from `typing` for a few extra versions right ?

Reply:
Should be, but now that we can use the classes directly this (dict instead of Dict) this should be the way forward.